code/modules/events/meateor_wave.dm

```diff
@@ -5,7 +5,7 @@
 	max_occurrences = 1
 
 /datum/round_event/meteor_wave/meaty/announce()
-	priority_announce("Meaty ores have been detected on collision course with the station.", "Oh Crap, Get The Mop.",'sound/AI/meteors.ogg')
+	priority_announce("Meaty ores have been detected on collision course with the station.", "Oh crap, get the mop.",'sound/AI/meteors.ogg')
```

Comment:
Rabble rabble grammar rabble someone else can revert it later if they don't like it